src/rdf-defs.js

```diff
@@ -0,0 +1,171 @@
+// @prefix dockalpha: <https://rdf.dock.io/alpha/2021#> .
+// @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+
+// id: dockalpha:mayClaim
+// type: rdf:Property
+// domain: rdf:Resource
+// range: dockalpha:ClaimFilter
+//
+// An entity that attests to a dockalpha:mayClaim statement grant the subject of that statement
+// authority to make any claims statisfying the object of the statment on the entities behalf.
+export const MAYCLAIM = 'https://rdf.dock.io/alpha/2021#mayClaim';
+
+// id: dockalpha:ANYCLAIM
+// type: dockalpha:ClaimFilter
+//
+// The "allow all" filter. dockalpha:ANYCLAIM accepts all triples.
+// All triples satisfy dockalpha:ANYCLAIM.
+export const ANYCLAIM = 'https://rdf.dock.io/alpha/2021#ANYCLAIM';
+
+// id: dockalpha:ClaimFilter
+// type: rdf:Class
+//
+// Filter over rdf claims.
+export const CLAIM_FILTER = 'https://rdf.dock.io/alpha/2021#ClaimFilter';
```

Comment:
This section is the start of an ontology definition like https://www.w3.org/1999/02/22-rdf-syntax-ns# .